server/utils/resident/index.test.ts

```diff
@@ -44,6 +44,24 @@ describe('residentsUtils', () => {
         },
       ])
     })
+
+    it('falls back to the resident show path when a tab has no explicit route', () => {
+      const placement = cas1SpaceBookingFactory.build()
+      const uknownLabel = 'label123'
+      const tabLabelsRecord = tabLabels as unknown as Record<string, { label: string }>
+      tabLabelsRecord.mystery = { label: uknownLabel }
+
+      const tabs = residentTabItems(placement, 'personal')
+      const mysteryTab = tabs.find(tab => tab.text === uknownLabel)
+
+      expect(mysteryTab).toEqual({
+        active: false,
+        href: `/manage/resident/${placement.person.crn}/placement/${placement.id}`,
+        text: uknownLabel,
+      })
+
+      delete tabLabelsRecord.mystery
+    })
```

Comment:
I don't think this test is useful actually. 
The `tabLabels` array is hooked on the `ResidentProfileTab` so all labels have to be present. You had to jump through casting hoops to fool the type checker so I don't how there could ever be a genuine error here. Perhaps I'm missing something? 😕